src/main/java/io/pravega/perf/PravegaTransactionWriterWorker.java

```diff
@@ -30,7 +31,7 @@
                                    PerfStats stats, String streamName, int eventsPerSec, boolean writeAndRead,
                                    ClientFactory factory, int transactionsPerCommit) {
 
-        super(sensorId, events, secondsToRun, isRandomKey,
+        super(sensorId, events, Integer.MAX_VALUE, secondsToRun, isRandomKey,
```

Comment:
Why for transactions flush is set to max?

Reply:
for transactions --transactionspercommit is used , so flush is not applicable for transactions.